rotkehlchen/serialization/deserialize.py

```diff
@@ -186,6 +186,10 @@ def deserialize_timestamp_from_kraken(time: Union[str, FVal, int]) -> Timestamp:
             raise DeserializationError(
                 f'Failed to deserialize {time} kraken timestamp entry',
             ) from e
+
+    if isinstance(time, float):
```

Comment:
This will work but a better change (as a diff on top of your commit is):

```diff
modified   rotkehlchen/serialization/deserialize.py
@@ -179,17 +179,15 @@ def deserialize_timestamp_from_kraken(time: Union[str, FVal, int, float]) -> Tim
 
     if isinstance(time, int):
         return Timestamp(time)
-    if isinstance(time, str):
+
+    if isinstance(time, (float, str)):
         try:
             return Timestamp(convert_to_int(time, accept_only_exact=False))
         except ConversionError as e:
             raise DeserializationError(
-                f'Failed to deserialize {time} kraken timestamp entry',
+                f'Failed to deserialize {time} kraken timestamp entry from {type(time)}',
             ) from e
 
-    if isinstance(time, float):
-        time = FVal(time)
-
     if isinstance(time, FVal):
         try:
             return Timestamp(time.to_int(exact=False))
```

That's because `convert_to_int` already takes `Union[FVal, bytes, str, int, float]`.